build.sbt

```diff
@@ -30,7 +30,7 @@ name := "ip-contributions"
 
 version := "1.0.0"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.8"
 
 crossScalaVersions := Seq("2.11.12", "2.12.4")
```

Comment:
This should probably be:
```scala
crossScalaVersions := Seq("2.11.12", "2.12.8")
```
to match the `2.12.8` above. `sbt` in its own `build.sbt` does something like
```scala
val scala211 = "2.11.12"
val scala212 = "2.12.8"

scalaVersion := scala212

crossScalaVersions := Seq(scala212, scala211)
```
to DRY